site/content/component/hono-server.md

```diff
@@ -19,24 +19,74 @@ The following table provides an overview of the configuration options the adapte
 | `HONO_SERVER_TRUST_STORE_PASSWORD`<br>`--hono.server.trustStorePassword` | no | - | The password required to read the contents of the trust store. |
+| `HONO_SERVER_TRUST_STORE_PATH`<br>`--hono.server.trustStorePath` | no  | - | The absolute path to the Java key store containing the CA certificates the Hono server uses for authenticating clients. The key store format can be either `JKS`, `PKCS12` or `PEM` indicated by a `.jks`, `.p12` or `.pem` file suffix. |
 
 The options only need to be set if the default value does not match your environment.
 
+### Port configuration
+
+Hono server supports a secure and an insecure listening port by means of configuration. This leverages the following configuration scenarios:
+
+- only one secure port (default)
+- only one insecure port
+- one secure AND one insecure port (dual port configuration)
+
+If both ports are not configured correctly or are disabled, the Hono server will fail to start. 
+
+#### Only one secure port
+This is the default configuration. It only needs the configuration of a keystore (`HONO_SERVER_KEY_STORE_*`) or PEM files (`HONO_SERVER_KEY_PATH` and `HONO_SERVER_CERT_PATH`).
+
+Hono server then opens the IANA default port number 5671 as TLS secured AMQP 1.0 port. 
+
+If `HONO_SERVER_PORT` is configured explicitly this port will be used instead. 
+
+To configure the port visibility in the network setup, set `HONO_SERVER_INSECURE_PORT_BIND_ADDRESS` explicitly. 
+
+#### Only one insecure port
+
+While in production scenarios it is almost certainly necessary that the Hono server communicates via the encrypted AMQP 1.0 port, a setup with an insecure (i.e. unencrypted) AMQP port sometimes is
+desirable.
+This may ease the debugging of AMQP 1.0 frames by using proxies or network sniffers that log the details while developing protocol adapters e.g.  
+
+To avoid accidental configuration of an insecure port this feature of Hono server must be explicitly enabled by  setting `HONO_SERVER_INSECURE_PORT_ENABLED` to true.
+
+Hono server then opens the IANA default port number 5672 as unsecured AMQP 1.0 port. 
+
+If `HONO_SERVER_INSECURE_PORT` is configured explicitly this port will be used instead. 
+If set to the IANA port number 5671 for secured AMQP 1.0 a warning is printed.
+
+
+#### One secure AND one insecure port (dual port configuration)
+ 
+In test setups and some production scenarios Hono server may be configured to open one secure AND one insecure port at the same time.
+ 
+This is achieved by configuring both ports correctly (see above). 
+If the same port is configured for both it is considered as misconfiguration and Hono server fails to start.
+
+Since the secure port may have different visibility in the network setup compared to the secure port, it has it's own binding address `HONO_SERVER_INSECURE_PORT_BIND_ADDRESS`. 
```

Comment:
Right, should have been  "....port may need different visibility...".
Corrected, and also the other points above.